org.scala-ide.sdt.debug.tests/src/org/scalaide/debug/internal/launching/RemoteConnectorTest.scala

```diff
@@ -208,9 +219,9 @@ class RemoteConnectorTest {
   /**
    * Check if it is possible to connect to a running VM that did not suspend.
    */
-  @Test(timeout = 5000)
+  @Ignore("Debugee cannot guarantee to wait for its Debugging")
+  @Test(timeout = 10000)
```

Comment:
This is for this ignored test, but I'll change it. Locally it does not need
it so long

2015-12-29 12:44 GMT+01:00 Simon Schäfer <notifications@github.com>:

> In
> org.scala-ide.sdt.debug.tests/src/org/scalaide/debug/internal/launching/RemoteConnectorTest.scala
> <https://github.com/scala-ide/scala-ide/pull/1043#discussion_r48535371>:
>
> > @@ -208,9 +219,9 @@ class RemoteConnectorTest {
> >    /**
> >     * Check if it is possible to connect to a running VM that did not suspend.
> >     */
> > -  @Test(timeout = 5000)
> > +  @Ignore("Debugee cannot guarantee to wait for its Debugging")
> > +  @Test(timeout = 10000)
>
> The configured timeout is still huge. Do you want to keep them this way?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scala-ide/scala-ide/pull/1043/files#r48535371>.
>